### PR TITLE
SN 1.5: de-Implification (or S-Implification)

### PR DESCRIPTION
**Pull request by ialarmedalien**
For ease of wrangling, this PR moves a chunk of code out of the Impl file and into its own class.

### CODE REVIEW

File: lib/StaticNarrative/StaticNarrativeImpl.py [hunk 1]
```diff
@@ -1,29 +1,23 @@
 # -*- coding: utf-8 -*-
-#BEGIN_HEADER
-import logging
-import os
-from StaticNarrative.exporter.exporter import NarrativeExporter
-from StaticNarrative.uploader.uploader import upload_static_narrative
-from StaticNarrative.narrative_ref import NarrativeRef
+# BEGIN_HEADER
```
Comment (ialarmedalien) [inline lib/StaticNarrative/StaticNarrativeImpl.py:2]: Ran black on the files without thinking... hence the templated text has also been formatted.

File: lib/StaticNarrative/StaticNarrativeImpl.py [hunk 2]
```diff
@@ -71,46 +55,19 @@ def create_static_narrative(self, ctx, params):
...
-        output = {
-            "static_narrative_url": static_url
-        }
-        self.logger.info(f"Finished creating Static Narrative {ref}")
-        #END create_static_narrative
+        # BEGIN create_static_narrative
+        snc = StaticNarrativeCreator(self.config)
+        params["user_id"] = ctx["user_id"]
+        params["token"] = ctx["token"]
+        output = snc.create_static_narrative(params)
```
Comment (ialarmedalien) [inline lib/StaticNarrative/StaticNarrativeImpl.py:62]: Everything on the left hand side has been moved with little or no change into the StaticNarrativeCreator package.

File: lib/StaticNarrative/StaticNarrativeImpl.py (hunk 1 above)
Comment (briehl) [reply lib/StaticNarrative/StaticNarrativeImpl.py:2]: Yikes. Is that going to break things? I seem to recall the template builder / rebuilder breaking when it sees a space after the hash.

Or is there an SDK version now that fixes that? I also remember some work being done around that.

**Review by briehl — APPROVED** [review]
Pretty straightforward. Looks good.

You might want to check if the reformatted templates break things when made again. I imagine it'll build and test just fine right now, though, so you can always fix that when it becomes a real problem.

File: lib/StaticNarrative/StaticNarrativeImpl.py (hunk 1 above)
Comment (ialarmedalien) [reply lib/StaticNarrative/StaticNarrativeImpl.py:2]: The SDK now accepts proper python formatting so it's not a problem. :D

**Pull request merged**